### ISSUE TITLE
Listen at specific times

### ISSUE THREAD

**Issue by cian-w** [MatteoGabriele/vue-speech#3]
Hey, is there anyway to get vue-speech to listen at specific times. For example when a user clicks a button start listening. Rather than listening once the page is loaded?

Thanks

**Comment by MatteoGabriele** [comment]
Not really, but I should integrate that kind of functionalities. For now you can just use an v-if on the component and show/hide it

**Comment by cian-w** [comment]
Yep doing that. Cool!
